.github/workflows/tests.yml

```diff
@@ -9,6 +9,14 @@ jobs:
       clickhouse:
         image: clickhouse/clickhouse-server:latest
         ports: [ '8123:8123' ]
+        env:
+          CLICKHOUSE_USER: default
+          CLICKHOUSE_PASSWORD: ""   # empty if your ClickHouse allows it
```

Comment:
How about this?
          CLICKHOUSE_PASSWORD: ""   # ClickHouse accepts an empty password